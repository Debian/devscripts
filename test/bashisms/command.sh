#!/bin/sh

command test
command -p test
command -v test # BASHISM
command -V test # BASHISM
