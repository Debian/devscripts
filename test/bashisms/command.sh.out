possible bashism in bashisms/command.sh line 5 ('command' with option other than -p):
command -v test # BASHISM
possible bashism in bashisms/command.sh line 6 ('command' with option other than -p):
command -V test # BASHISM
